modules/howtos/examples/ManagingConnections.java

```diff
@@ -166,6 +166,13 @@ public static void main(String... args) {
       // end::tls[]
     }
 
+    {
+      // tag::tls-no-trust[]
+      ClusterEnvironment env = ClusterEnvironment.builder()
```

Comment:
Instead of showing users how to create their own ClusterEnvironment (which his something I'd personally like to discourage, since it means the user is responsible for shutting it down, and the environment won't honor the connection string scheme or parameters), would it make sense to show an example that uses a connection string that starts with "couchbases://" ?